### DISCUSSION TITLE
Few Questions

### DISCUSSION THREAD

**Discussion by cloudcompute** [mlrun/mlrun#3314] [Q&A] [answered]
Hi

MLRun looks great. Some questions relating to it:

a. Any companies who are using it in `production`?

b. Can we `integrate` it with a `vector search engine` like Vespa, Weaviate?

c. Taken from Iguazio's site: Its `managed service` provides this feature: 

> Real-time multi-modal data layer.. The data layer provides fast, secure and shared access to real-time and historical data including NoSQL, SQL, time series and files.

What does it mean, especially the word `shared`?

d. `How flexible` is MLRun.. for example, it possible to override a subset of its functionality for example, feature store with ours like Feast Or Model Serving with Vespa model serving?

e. Can we use `Deltalake as an offline store`?

f. Is there any example that demonstrates the e-commerce's `Personalised recommendations` feature?

Thanks

**Comment by gilad-shaham** [comment, marked as answer]
It should be possible to integrate MLRun with additional services, but we haven't done the specific integrations you mentioned, because the need never came up.
WRT production, yes, the commercial offering which includes MLRun in its core does have clients who use MLRun in production. The data layer is not part of MLRun, it is part of the commercial offering, but the open source MLRun implementation integrates with open source projects such as Kafka and Redis.

**Comment by bobvanluijt** [comment]
Super interesting project; if we can help with Weaviate. Feel free to join our [Slack](https://weaviate.io/slack) or get started [here](https://weaviate.io/developers/weaviate/current/).

**Reply by cloudcompute** [reply]
Thank you @bobvanluijt for letting me know that it is possible to integrate MLRun wioth Weaviate.
